operations/nodes.rst

```diff
@@ -46,13 +46,25 @@ Removing nodes are as simple as:
 
    $ juju remove-unit slurmd/4
 
+.. warning::
+
+   This is a destructive operation. If the node is running a job, this job will
+   be killed and SLURM will not requeue it. It is the system administrator's
+   responsibility to ensure that the node can be safely removed.
+
+It is recommended to first *drain* nodes before removing them from the cluster
+(see :ref:`draining-nodes`), as to not destroy running jobs.
```

Comment:
`as to not destroy running jobs.` to `for not destroying running jobs.`. what do you think?